compiler/element/frontend/parser.py

```diff
@@ -2,14 +2,24 @@
 import pathlib
 
 from lark import Lark
+from lark.indenter import Indenter
+
+
+class AppNetIndenter(Indenter):
+    NL_type = "_NEWLINE"
+    INDENT_type = "_INDENT"
+    DEDENT_type = "_DEDENT"
+    OPEN_PAREN_types = []
+    CLOSE_PAREN_types = []
+    tab_len = 4
 
 
 class ElementParser:
     def __init__(self):
         cwd = pathlib.Path(__file__).parent
-        grammar = open(os.path.join(cwd, "element.lark"), "r").read()
+        grammar = open(os.path.join(cwd, "python_style_grammar.lark"), "r").read()
```

Comment:
I think we can just call it `element.lark`?

Reply:
I think we can call this `element.lark` and just keep a copy of the old one for reference.